src/custom/pages/Claim/index.tsx

```diff
@@ -431,33 +438,43 @@ export default function Claim() {
                   {sortedClaimData.map(({ index, type, amount }) => {
                     const isFree = isFreeClaim(type)
                     const currency = typeToCurrencyMap[type] || ''
-                    const vCowPrice = typeToPriceMap[type]
+                    const vCowPrice = typeToPriceMap.get(type)
                     const parsedAmount = parseClaimAmount(amount, chainId)
-                    const cost = vCowPrice * Number(parsedAmount?.toSignificant(6))
+                    const cost = vCowPrice && vCowPrice * Number(parsedAmount?.toSignificant(6))
```

Comment:
I know, not from this PR, but just to keep in mind to not work with Number instances due to precision etc etc

Reply:
yeah not this PR so let's tackle this badboi later